candidates/models/db.py

```diff
@@ -51,7 +51,7 @@ class LoggedAction(models.Model):
     person = models.ForeignKey(Person, blank=True, null=True)
     action_type = models.CharField(max_length=64)
     popit_person_new_version = models.CharField(max_length=32)
-    created = models.DateTimeField(auto_now_add=True)
+    created = models.DateTimeField(auto_now_add=True, db_index=True)
```

Comment:
Sorry about that I think I forgot to add it and then did `commit --amend` on the wrong commit. Added to the first on now.